src/components/table/table-cell/table-cell.style.js

```diff
@@ -103,6 +104,10 @@ function applyModernInputStyling(isTextArea, inputHeight, fontSize) {
       padding-top: 0px;
       padding-bottom: 0px;
     }
+    
+    .carbon-icon {
+      font-size: 13px !important;
+    }
```

Comment:
could it be done with `&&.carbon-icon` instead of using the `!important` keyword?

Reply:
Yeah good point I'll give it a try

Reply:
Made the changes suggested 👍